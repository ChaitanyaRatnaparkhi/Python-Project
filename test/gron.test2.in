{
  "person": {
    "name": "Alice",
    "age": 30,
    "address": {
      "city": "Wonderland",
      "zip": "12345"
    }
  }
}