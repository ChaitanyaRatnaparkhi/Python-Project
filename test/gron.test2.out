json = {}
json.person = {{}}
json.person.name = "Alice"
json.person.age = 30
json.person.address = {{}}
json.person.address.city = "Wonderland"
json.person.address.zip = "12345"

